lang/cs/Org.Apache.REEF.Network/Group/Operators/IOperatorSpec.cs

```diff
@@ -26,6 +26,6 @@ namespace Org.Apache.REEF.Network.Group.Operators
     /// </summary>
     public interface IOperatorSpec
     {
-        IConfiguration Configiration { get; }
```

Comment:
This one probably needs to go though obsolete path as well, since it will be public in BroadcastOperatorSpec